src/tasks/disable-import-audit-processor/handler.js

```diff
@@ -36,7 +36,15 @@ export async function runDisableImportAuditProcessor(message, context) {
     organizationId,
     importTypes,
     auditTypes,
+    scheduledRun,
   });
+
+  if (scheduledRun) {
+    log.info('Scheduled run detected - skipping disable of imports and audits');
+    await say(env, log, slackContext, `:information_source: Scheduled run detected for site ${siteUrl} - skipping disable of imports and audits`);
```

Comment:
can you add site_id as well? I've seen that we sometimes onboard different pathes of website as a separate site_id